lib/sisimai/bite/email/google.rb

```diff
@@ -0,0 +1,320 @@
+module Sisimai::Bite::Email
+  module Google
+    # Sisimai::Bite::Email::Google parses a bounce email which created by Gmail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Google.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/[@]googlemail[.]com[>]?\z/,
+        :subject => %r/Delivery[ ]Status[ ]Notification/,
+      }
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->